modules/pam_faillock/pam_faillock.c

```diff
@@ -620,7 +620,20 @@ faillock_message(pam_handle_t *pamh, struct options *opts)
 		if (left > 0) {
 			left = (left + 59)/60; /* minutes */
 
-			pam_info(pamh, _("(%d minutes left to unlock)"), (int)left);
+#if defined HAVE_DNGETTEXT && defined ENABLE_NLS
+			pam_info(
+				pamh,
+				dngettext(PACKAGE,
+					"(%d minute left to unlock)",
+					"(%d minutes left to unlock)",
+					(int)left),
+				(int)left);
+#else
+			if (left == 1)
+				pam_info(pamh, _("(%d minute left to unlock)"), (int)left);
+			else
+				pam_info(pamh, _("(%d minutes left to unlock)"), (int)left);
```

Comment:
Please add a comment for translators:
/* TRANSLATORS: only used if dngettext is not supported.  */

Reply:
Probably also for line 633?